wsmaster/che-core-sql-schema/src/main/resources/che-schema/5.20.0/1__add_machine_env.sql

```diff
@@ -0,0 +1,20 @@
+--
+-- Copyright (c) 2012-2017 Red Hat, Inc.
+-- All rights reserved. This program and the accompanying materials
+-- are made available under the terms of the Eclipse Public License v1.0
+-- which accompanies this distribution, and is available at
+-- http://www.eclipse.org/legal/epl-v10.html
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+-- Environment machine env ----------------------------------------------
+CREATE TABLE externalmachine_env (
```

Comment:
Can you elaborate on why external?

Reply:
Actually, I don't know. I do in this way to be the same with existing tables https://github.com/eclipse/che/blob/36777244e994f1c822a86f27656fd37a82c52ef7/wsmaster/che-core-sql-schema/src/main/resources/che-schema/5.0.0-M8/1__init.sql#L271-L300

Reply:
Can we avoid that historical thing